oojs.js

```diff
@@ -4,9 +4,30 @@
   // A favorites property that is initialized at 0. This cannot be set by user input.
   // An addSong method that adds a song to the songs property.
 
+  var Playlist {
+    constructor(title, songs, favorites){
+      this.title = title;
+      this.songs = songs;
+      this.favorites= 0;
+      }
+    addSong: function() {
+      this.songs ++;
+      return 'whine up!';
+    }
+  }
+
 // Create an instance of the Playlist class.
+  let Dancehall = new Playlist('Danceup', ('Bubble Time', 'Mash up de place', 0));
 
 // Create an Album class that inherits from Playlist. It should also...
   // Have an artist property that is determined by some input.
 
+  class Album extends Playlist {
+    constructor(title, songs, favorites, artist){
+      super(title, songs, favorites)
+      this.artist = artist;
+    }
+  }
+
```

Comment:
favorites is supposed to not be set by user input but this is totally correct otherwise!